include/libkrun.h

```diff
@@ -47,23 +46,27 @@ int32_t krun_set_log_level(uint32_t level);
  * Initializes logging for the library.
  *
  * Arguments:
- *  "target_fd" - File descriptor to write log to. Note that using a file descriptor pointing to a regular file on
- *                filesystem might slow down the VM.
- *                Use KRUN_LOG_TARGET_DEFAULT to use the default target for log output (stderr).
+ *  "target_fd" - File descriptor to write log to. Note that using a file
+ * descriptor pointing to a regular file on filesystem might slow down the VM.
+ *                Use KRUN_LOG_TARGET_DEFAULT to use the default target for log
+ * output (stderr).
  *
- *  "level"     - Level is an integer specifying the level of verbosity, higher number means more verbose log.
- *                The log levels are described by the constants: KRUN_LOG_LEVEL_{OFF, ERROR, WARN, INFO, DEBUG, TRACE}
+ *  "level"     - Level is an integer specifying the level of verbosity, higher
+ * number means more verbose log. The log levels are described by the constants:
+ * KRUN_LOG_LEVEL_{OFF, ERROR, WARN, INFO, DEBUG, TRACE}
  *
- *  "style"     - Enable/disable usage of terminal escape sequences (to display colors)
- *                One of: KRUN_LOG_STYLE_{AUTO, ALWAYS, NEVER}.
+ *  "style"     - Enable/disable usage of terminal escape sequences (to display
+ * colors) One of: KRUN_LOG_STYLE_{AUTO, ALWAYS, NEVER}.
  *
  *  "options"   - Bitmask of logging options, use 0 for default options.
- *                KRUN_LOG_OPTION_NO_ENV to disallow environment variables to override these settings.
+ *                KRUN_LOG_OPTION_NO_ENV to disallow environment variables to
+ * override these settings.
  *
  * Returns:
  *  Zero on success or a negative error number on failure.
  */
-int32_t krun_init_log(int target_fd, uint32_t level, uint32_t style, uint32_t options);
+int32_t krun_init_log(int target_fd, uint32_t level, uint32_t style,
```

Comment:
Seems like there' some unrelated changes other than adding `krun_set_firmware` in this commit. Not a problem, just making sure you're aware (perhaps this was from some C formatting tool?)